protos/flyteidl/admin/common.proto

```diff
@@ -237,14 +237,46 @@ message Annotations {
     map<string, string> values          = 1;
 }
 
+// Security context fields to define privilege and access control settings
+// In the future a mode (e.g. OVERRIDE, APPEND, etc) can be defined
+// to specify how to merge security context defined at registration and execution time.
+message SecurityContext {
+    // Map of security context values
+    map<string, string> values          = 1;
+}
+
+// Config values to define privilege and access control settings
+// In the future a mode (e.g. OVERRIDE, APPEND, etc) can be defined
+// to specify how to merge configs defined at registration and execution time.
+message Config {
+    // Map of custom config values
+    map<string, string> values          = 1;
+}
+
 // Defines permissions associated with executions.
+// Deprecated
 message AuthRole {
   oneof method {
     string assumable_iam_role         = 1;
     string kubernetes_service_account = 2;
   }
 }
 
+// Defines values that can be overriden at registration/runtime
+message Overrideables {
```

Comment:
thoughts on naming this something more specific, like ExecutionOverrideables?

Reply:
Sure but it is part of LP also.

Reply:
good point @EngHabu any thoughts?